bitnami/nginx-ingress-controller/templates/default-backend-configmap.yaml

```diff
@@ -0,0 +1,17 @@
+{{- if and (.Values.defaultBackend.enabled) (.Values.defaultBackend.serverBlockConfig) -}}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ template "nginx-ingress-controller.defaultBackend.fullname" . }}
+  namespace: {{ .Release.Namespace | quote }}
+  labels: {{- include "common.labels.standard" . | nindent 4 }}
+    app.kubernetes.io/component: default-backend
+    {{- if .Values.commonLabels }}
+    {{- include "common.tplvalues.render" (dict "value" .Values.commonLabels "context" $) | nindent 4 }}
+    {{- end }}
+  {{- if .Values.commonAnnotations }}
+  annotations: {{- include "common.tplvalues.render" ( dict "value" .Values.commonAnnotations "context" $ ) | nindent 4 }}
+  {{- end }}
+data:
+  defaultBackend.conf: {{ toYaml .Values.defaultBackend.serverBlockConfig | indent 2 }}
```

Comment:
Check my comment on the `values.yaml` file. It's also better if we use the `"common.tplvalues.render"` function instead of the `toYaml` function to add support for templating.